test/mic-check.html

```diff
@@ -0,0 +1,115 @@
+<html>
+
+<head>
+  <link rel="stylesheet" href="./styles.css" />
+</head>
+
+<body>
+  <button onclick="start()">Start!</button>
+  <div id="messages"></div>
+  🎤
+  <button onclick="recordingStart()">⏺</button>
+  <button onclick="recordingStop()">⏹</button>
+</body>
+<script type="module">
+  import {
+    Playthrough,
+    Speaker,
+            createPlaythroughToken,
+            createConversation,
+          } from "../dist/index.js";
+
+    import {
+      setGlobalBaseUrl,
+    } from "../dist/api.js";
+
+    // TEST PARAMETERS
+    const storyId = 2;
+    const version = -1;
+      const userToken = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoiYXV0aGVudGljYXRlZCIsInVzZXJfaWQiOiI3MDA5ZDNmMS1iYTE0LTRhMDEtYTJhOC1hNjA5ZTdlN2NkOGYiLCJpYXQiOjE2OTE2NzIwMDIsImV4cCI6MTY5NDI2NDAwMiwiYXVkIjoicG9zdGdyYXBoaWxlIiwiaXNzIjoicG9zdGdyYXBoaWxlIn0.DMu6QJjOu7yISGQ-ajM0sua9VON9zvxNyEdrKB4uq2I";
```

Comment:
not a massive deal to have this here as nothing sensitive inside it but might want to consider redacting, or maybe turning all these config things into env vars: https://en.parceljs.org/env.html

Reply:
I don't think it's worth the effort, adds a layer of complexity and values will need to be manually input every time anyway